### PR TITLE
fix the 2 broken linters

### PR DESCRIPTION
**Pull request by fchapoton**
This repairs the two broken linters.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/rings/polynomial/polynomial_zz_pex.pyx [hunk 1]
```diff
@@ -547,7 +547,7 @@ cdef class Polynomial_ZZ_pEX(Polynomial_template):
     def inverse_series_trunc(self, prec):
         r"""
         Compute and return the inverse of self modulo `x^prec`.
```
Comment (dcoudert) [inline src/sage/rings/polynomial/polynomial_zz_pex.pyx]: I assume it should be `x^{prec}` here.

Comment (fchapoton) [reply src/sage/rings/polynomial/polynomial_zz_pex.pyx]: voilà, voilà

**Comment by dcoudert** [comment]
failing doctest
```
sage -t --long --random-seed=286735480429121101562228604801325644303 sage/dynamics/arithmetic_dynamics/dynamical_semigroup.py
**********************************************************************
File "sage/dynamics/arithmetic_dynamics/dynamical_semigroup.py", line 933, in sage.dynamics.arithmetic_dynamics.dynamical_semigroup.DynamicalSemigroup.__mul__
Failed example:
    f*g
Expected:
    Traceback (most recent call last):
    ...
    TypeError: can only multiply dynamical semigroups with other dynamical semigroups of the same type
Got:
    <BLANKLINE>
    Traceback (most recent call last):
      File "/sage/src/sage/doctest/forker.py", line 709, in _run
        self.compile_and_execute(example, compiler, test.globs)
      File "/sage/src/sage/doctest/forker.py", line 1144, in compile_and_execute
        exec(compiled, globs)
      File "<doctest sage.dynamics.arithmetic_dynamics.dynamical_semigroup.DynamicalSemigroup.__mul__[46]>", line 1, in <module>
        f*g
        ~^~
      File "/sage/src/sage/dynamics/arithmetic_dynamics/dynamical_semigroup.py", line 967, in __mul__
        raise ValueError("left dynamical semigroup's domain must equal right dynamical semigroup's codomain")
    ValueError: left dynamical semigroup's domain must equal right dynamical semigroup's codomain
**********************************************************************
1 item had failures:
   1 of  62 in sage.dynamics.arithmetic_dynamics.dynamical_semigroup.DynamicalSemigroup.__mul__
    [398 tests, 1 failure, 5.31 s]
```

**Comment by EnderWannabe** [comment]
Changes look good to me.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**